app/models/issue.rb

```diff
@@ -25,8 +26,32 @@ def absent?
     new_record? && !detected?
   end
 
-  def fixed?
-    !detected?
+  def closed?
+    closed_at.present?
+  end
+
+  def open?
+    !closed?
+  end
+
+  def fix!
+    return if closed?
+
+    self.close_result ||= :fixed
+    self.closed_at = Time.zone.now
+    save!
+  end
+
+  def gone!
+    return if closed?
+
+    self.close_result = :gone
```

Comment:
I see that `gone!` overwrites the `close_result` while `fix!` does not. Is it expected? I also see that there's no `spec/commands/issues/close_issue_spec.rb`, maybe that unit test could catch this (if there was a bug here at all).

Reply:
You are right, I've changed the `close_result` assignment in the `CloseIssue` to keep it consistent.